services/wfm/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/update/actions/RevertNewRulesAction.java

```diff
@@ -92,4 +94,21 @@ protected void perform(State from, State to, Event event, FlowUpdateContext cont
         stateMachine.saveActionToHistory(
                 "Commands for removing new rules and re-installing original ingress rule have been sent");
     }
+
+    private SpeakerRequestBuildContext getSpeakerRequestBuildContext(FlowUpdateFsm stateMachine) {
```

Comment:
Is it a copy of method in RemoveOldRulesAction?

Reply:
moved to common method